elastic/kibana
# Fix: Consider all summary pings to determine if monitor is only fleet managed.

Fixes https://github.com/elastic/kibana/issues/138862 
Fixes https://github.com/elastic/kibana/issues/140768

## Summary

- Fixes the behavior where **Run now** button  on Uptime -> Monitor Management -> Add/Edit page would try to initiate a pre-flight (`run_once`) test for both public and private locations, in which case, the test for private location never finishes. Also some tweaks are applied to the **Run now** tooltip (popover).
- Fixes the behavior on Uptime Overview page where running an on-demand test (**Test now**, the button with _play_ icon) for a monitor configured for both public and private locations would disable the **Test now** button on on-demand test completion.

### How to test
1. Set up a cluster so that you are able to create and run a monitor on at least one public and at least one private location (Only a shallow private location with no connected agent may also work for this test but a thorough e2e test is recommended). Or you may use the [deployment](https://p.elstc.co/paste/7LGjA43F#dakuloOk7IoQSREBTPnP+AEYRFEww9j4HGMnXRGXtdH) if it's still live.
2. Create the monitor via Uptime -> Monitor Management and test the **Test now** behavior before saving and make sure the pre-flight test is only ran for public locations.
3. Save the monitor and wait for the results to get ingested.
4. Goto Uptime -> Overview page and make sure the monitor appears there.
5. Test the on-demand test functionality via the **Test now** button (the button with _play_ icon) and make sure the button stays enabled after the test completes.
<img width="1714" alt="Screenshot 2022-09-28 at 20 23 42" src="https://user-images.githubusercontent.com/2748376/192878297-1f58d8c2-0979-4933-9b2b-41f1a7f6649c.png">

<img width="1714" alt="Screenshot 2022-09-28 at 21 40 38" src="https://user-images.githubusercontent.com/2748376/192878538-984bf733-1320-4696-9f8f-9ffba26be8af.png">

<img width="1711" alt="Screenshot 2022-09-28 at 21 43 46" src="https://user-images.githubusercontent.com/2748376/192878560-1c8dcfc9-dbca-4611-bb51-0fc3454354bc.png">
